### ISSUE TITLE
Add ng-toolkit/serverless  ENOVERSIONS

### ISSUE THREAD

**Issue by glancey** [maciejtreder/ng-toolkit#596]
## I'm submitting a...

<pre><code>
[ x] Bug report  
[ ] Feature request
[ ] Documentation issue or request
</code></pre>

## In package
<pre><code>
[ ] @ng-toolkit/init
[ x] @ng-toolkit/serverless
[ ] @ng-toolkit/universal
[ ] @ng-toolkit/pwa
[ ] @ng-toolkit/firebug
</code></pre>

## Current behavior
UPDATE package.json (2300 bytes)
npm ERR! code ENOVERSIONS
npm ERR! No valid versions available for webpack-cl

## Minimal reproduction of the problem with instructions
AWS user added and the following commands run locally.

serverless config credentials --key ACCESSKEY --secret SECRETKEY
ng new myApp
cd myApp
ng add @ng-toolkit/serverless --provider aws

## Environment

<pre><code>
Angular version: 7.2.0

- Node version: 8.9.4
- Platform:  MAC

Others:

</code></pre>

**Comment by YannicEl** [comment]
same problem

**Comment by adimango** [comment]
Same problem:

```npm ERR! code ENOVERSIONS
npm ERR! No valid versions available for webpack-cl

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.13.8
@angular-devkit/build-angular     0.13.8
@angular-devkit/build-optimizer   0.13.8
@angular-devkit/build-webpack     0.13.8
@angular-devkit/core              7.3.3
@angular-devkit/schematics        7.3.3
@angular/cdk                      7.3.3
@angular/cli                      7.3.8
@angular/material                 7.3.3
@ngtools/webpack                  7.3.8
@schematics/angular               7.3.3
@schematics/update                0.13.8
rxjs                              6.3.3
typescript                        3.2.4
webpack                           4.29.0

```

node: v11.13.0

**Comment by KingDarBoja** [comment]
Closed as fixed at #606
